src/layout/Home.js

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Row from 'react-bootstrap/Row';
+import Container from 'react-bootstrap/Container';
+import InsertionForm from './InsertionForm';
 
 function Home() {
-  return <div>Home page</div>;
+  const [records, setRecords] = useState([]);
+  function addIncome(amount, category, description) {
+    const prevRecords = records.slice();
+    prevRecords.push({
+      type: 'Income',
+      amount: amount,
```

Comment:
JS object setting of type `{propertyName: PropertyName}` can be shorthanded to just `{propertyName}`  (es6 syntax). Like so: `{
      type: 'Income',
      amount,
      category,
      description,
    }`
This will pass linter check here.